src/osdc/Filer.cc

```diff
@@ -453,6 +454,7 @@ void Filer::_do_truncate_range(TruncRange *tr, int fin)
     tr->oncommit->complete(0);
     trl.unlock();
     delete tr;
+    tr = nullptr;
```

Comment:
Both pr and tr are raw pointers. Setting them to nullptr has no effect in this case... Or I'm missing something?

Reply:
no, i don't think you are missing anything. otherwise, i am missing something...